bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/common/CommerceExperienceFragmentContainer.java

```diff
@@ -0,0 +1,40 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2022 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.commerce.core.components.models.common;
+
+import javax.annotation.Nullable;
+
+import org.apache.sling.api.resource.Resource;
+
+import com.adobe.cq.wcm.core.components.models.Component;
+
+/**
+ * Interface for an Experience Fragment container to be used in a
+ * {@code ProductList}
+ */
+public interface CommerceExperienceFragmentContainer extends Component {
+
+    @Nullable
```

Comment:
Same here, can this be null? Or phrased differently: Should a `CommerceExperienceFragmentContainer` where this is null?